lib/src/Status/OSNotification.mm

```diff
@@ -55,6 +57,17 @@ - (instancetype)initOSNotification:(OSNotification*) instance;
     [notification release];
 }
 
+void OSNotification::showIconBadgeNotificationMacOs(int allMentions)
```

Comment:
@endulab we won't display badge label only in case of mentions, but most likely in other cases like as well like replies, new contact request and so... maybe `setIconBadgeValue(int value)` is more appropriate name cause looking at the body of this function it actually sets the value of the icon's badge.

Please rename also on other places `allMentions` and `mentions` to `value` or `notificationsNumber` just to have more general naming.

Reply:
+1

Reply:
@saledjenic I changed "mentions" to "notificationsCount"